core/src/main/java/org/opensearch/sql/expression/function/BuiltinFunctionRepository.java

```diff
@@ -69,6 +66,11 @@ public void register(String namespace, FunctionResolver resolver) {
   }
 
 
+  public FunctionImplementation compile(BuiltinFunctionName functionName,
```

Comment:
what is the difference with `FunctionImplementation compile(FunctionName functionName, List<Expression> expressions)`? Should we only keep one interface?

Reply:
Removed in e7ccfc2bd2cc20c2c581210a58b5afda3ccf4d50